src/rules/color-named/__tests__/index.js

```diff
@@ -286,3 +286,37 @@ testRule(rule, {
     column: 22,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [ "never", { ignore: ["inside-function"] } ],
+
+  accept: [ {
+    code: "a { color: #000; }",
+  }, {
+    code: "a { color: gray(70%); }",
+  }, {
+    code: "a { background-image: url(red); }",
+  }, {
+    code: "a { color: map-get($colour, blue(60%)); }",
+  }, {
+    code: "a { color: map-get($colour, blue); }",
+  } ],
+
+  reject: [ {
+    code: "a { color: map-get($colour, blue), rebeccapurple; }",
```

Comment:
I say keep it. If you'd like you could use `background` and that would be valid, right?